salt/orchestrate/aws/map_templates/edx.yml

```diff
@@ -10,10 +10,11 @@
 {% set num_subnets = subnetids|length %}
 
 edx:
-  {% for app_type, num_instances in app_types.items() %}
-  {% for id_num in range(num_instances.edx) %}
+  {% for app_type, settings in app_types.items() %}
```

Comment:
Where is `app_types` defined?

Reply:
It gets passed in as a context object from the orchestrate state

Reply:
There is no longer a need for [VPC_RESOURCE_SUFFIX](https://github.com/mitodl/salt-ops/blob/d5a0dcd7359b7e826c29a24a9c40f416cf808e0c/salt/orchestrate/aws/map_templates/edx.yml#L1) correct?

Reply:
Correct